key/aziot-keyd/src/lib.rs

```diff
@@ -121,6 +121,26 @@ impl Api {
         Ok(handle)
     }
 
+    pub fn move_key_pair(&mut self, from: &str, to: &str, user: libc::uid_t) -> Result<(), Error> {
+        // Require the caller to be authorized to modify both keys.
```

Comment:
Sure, but that adds extra steps. Right now, the caller needs to know the key IDs to use this API. If we change this to key handle, the caller needs to know key handle, get key ID, then pass key handle to this API, which converts it back to key ID. Are the extra steps necessary?

Reply:
For something which has access to the key, it's extra steps. For something which only has the handle, it's the *only* way.

All other actions that use a key take just the handle, so why not this one?

Reply:
Ah, but I guess it would be unusual for a user to configure one UID have access to the `from` key and another to have access to the `to` key. So if the same UID has access to both, then taking `from` as a handle is indeed pointless.

So this is fine as is.